### PR TITLE
Right Click

### LINKED ISSUE #2: Right click and report a HTML element
The user needs to be able to right click on an HTML element and report it as violating the citizenship standards 

The report needs to include enough data to ID the HTML element(s), the website it was from
If scoring users based on their trustworthiness, need to include a UserID as well

Would be nice if they could select what type of violation it was
EX: 
- Paywall in the middle of article 
- Video with sound
- Externally controlled Ads

### PR DESCRIPTION
**Pull request by TheENGR**
image

### CODE REVIEW

File: src/rightClick.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+chrome.runtime.onInstalled.addListener(function() {
```
Comment (TheENGR) [inline src/rightClick.js]: Make a query that asks for these when a new window is created
https://developer.chrome.com/docs/extensions/reference/windows/#event-onCreated

File: src/rightClick.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	});
+});
+
+//https://developer.chrome.com/docs/extensions/mv3/migrating_to_service_workers/#state
+chrome.contextMenus.onClicked.addListener(function(info, tab) {
+	//handle context menu actions
+	console.log('CLICKED SOMETHING')
+	console.log(info)
+	console.log(tab)
+	console.log('LOAD')
```
Comment (TheENGR) [inline src/rightClick.js]: Don't log these

File: src/rightClick.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+	chrome.storage.local.get(["htmlElement"], ({ htmlElement }) => {
+		console.log(htmlElement)
```
Comment (TheENGR) [inline src/rightClick.js]: Send shit to the Lambdas

File: src/rightClick.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+	});
+})
+
+chrome.runtime.onMessage.addListener((message) => {
```
Comment (TheENGR) [inline src/rightClick.js:53]: Add comments

File: src/rightClick.js [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+	const htmlElement = await chrome.storage.local.get(["htmlElement", "instanceID"], storage => {
+		const infraction = {
+			reporter: storage.instanceID,
+			url: info.pageUrl,
+			type: info.menuItemId,
+			content: storage.htmlElement
+		};
+		console.log(infraction)
+		// TODO: SEND THIS TO THE INFRACTION LOGGING API
+		//await fetch(`https://cyber-citizenship-coefficient.com/infraction`,
```
Comment (TheENGR) [inline src/rightClick.js:36]: @ajaxx 
Once we have your [PR](https://github.com/Cyberspace-Citizenship-Coefficient/AWS-Lambdas/pull/17) done we can send data in

**Review by appasalvi — APPROVED** [review]

**Review by ajaxx — APPROVED** [review]

**Pull request merged**